ArduPlane/servos.cpp

```diff
@@ -968,6 +968,23 @@ void Plane::landing_neutral_control_surface_servos(void)
  
 }
 
+/*
+  sets rudder/vtail , and elevon to indicator positions that we are in a rudder arming waiting for neutral stick state
+*/
+void Plane::indicate_waiting_for_rud_neutral_to_takeoff(void)
+{
+    if (!seen_neutral_rudder)  {
```

Comment:
In general i'm really not a fan of this, there is lots of potential starting to fly before this has cleared, even in manual modes. 

If my vehicle is unstable in yaw and requires some yaw output to fly it could now crash on takeoff acro for example.

If this were limited to only the case where the flight mode is waiting for neutral rudder then it might be OK. Maybe checking throttle surpession would be enough, but really I would add a new bool "waiting_neutral_rudder" to be set in the two takeoff start checks.

Reply:
yes, I think this is a problem...will add different bool